src/main/scala/com/danielasfregola/twitter4s/entities/GeoBoundingBox.scala

```diff
@@ -0,0 +1,6 @@
+package com.danielasfregola.twitter4s.entities
+
+final case class GeoBoundingBox(southwest: GeoCoordinate, northeast: GeoCoordinate) {
+  def toLngLatPairs: Seq[(Double, Double)] =
```

Comment:
I believe this could just be:
```
def toLngLatPairs: (Double, Double)
```
since the sequence would only contain only exactly 1 element

Reply:
The sequence would contain 2 elements since it has 2 coordinates, so its a 

    ((Double, Double),(Double, Double))

Reply:
Apologies, you are absolutely right! What you have done is the right thing 👍 

Please, ignore my comment